README.md

```diff
@@ -48,25 +97,84 @@ To disable this, you can use the `--quiet` flag to prevent output from going to
 `--no-log` flag to prevent output from going to the log file.
 
 ```
-$ topos-playground start --quiet
+$ topos-playground clean --quiet
+```
+
+### Development
+
+To contribute to the development of the playground, [fork](https://github.com/topos-protocol/topos-playground/fork) the repository.
+
+Do your work within a branch on your fork. When it is ready, create a Pull Request that clearly explains the reasoning behind your code changes, what they do, and anything that we should be aware of when testing your code.
+
+### Discussion
+
+For community help or discussion, you can join the Topos Discord server:
+
+[https://discord.gg/zMCqqCbGMV](https://discord.gg/zMCqqCbGMV)
+
+### Troubleshooting
```

Comment:
The playground does not start clean on my machine. `npx @topos/playground start --verbose` ends with:

```
Running the ERC20 messaging infra...
    🏃 Running command: source /Users/aggron/.local/share/topos-playground/.env.secrets && cd /Users/aggron/.local/share/topos-playground/local-erc20-messaging-infra && docker compose up -d
   ❗ Error
   Find the full log file in /Users/aggron/.local/state/topos-playground/logs/log-2d9950e3-14dc-4661-b7c0-245df70b9201.log
```

The log says:

```
info: error during connect: Post "http://%2FUsers%2Faggron%2F.colima%2Fdocker.sock/v1.42/containers/8ef43a7e88f4d889fa9cc4be15f4973e83db725bcceb527bbb74a28932011bc7/start": EOF
info: 
error: ❗ Error
```

Running the last command manually I see this:

```
➜  ~ source /Users/aggron/.local/share/topos-playground/.env.secrets
➜  ~ cd /Users/aggron/.local/share/topos-playground/local-erc20-messaging-infra
➜  local-erc20-messaging-infra git:((v0.1.6)) docker compose up -d
[+] Running 18/18
 ✔ Container contracts-init    Exited                                                                                                                                    0.0s
 ✔ Container topos-init        Exited                                                                                                                                    0.0s
 ✔ Container infra-tce-boot-1  Healthy                                                                                                                                   0.0s
 ✔ Container incal-init        Exited                                                                                                                                    0.0s
 ✔ Container topos-node-1      Healthy                                                                                                                                   0.0s
 ✔ Container topos-node-3      Started                                                                                                                                   0.0s
 ✔ Container topos-node-2      Started                                                                                                                                   0.0s
 ✔ Container infra-tce-peer-2  Started                                                                                                                                   0.0s
 ✔ Container incal-node-3      Started                                                                                                                                   0.0s
 ✔ Container incal-node-4      Started                                                                                                                                   0.0s
 ✔ Container infra-tce-peer-4  Started                                                                                                                                   0.0s
 ✔ Container infra-tce-peer-1  Started                                                                                                                                   0.0s
 ✔ Container infra-tce-peer-3  Started                                                                                                                                   0.0s
 ✔ Container incal-node-1      Healthy                                                                                                                                   0.0s
 ✔ Container incal-node-2      Started                                                                                                                                   0.0s
 ✘ Container contracts-topos   service "contracts-topos" didn't complete successfully: exit 137                                                                          0.0s
 ✘ Container contracts-incal   service "contracts-incal" didn't complete successfully: exit 1                                                                            0.0s
 ✔ Container topos-node-4      Started                                                                                                                                   0.0s
service "contracts-topos" didn't complete successfully: exit 137
```
```

Reply:
Looping back to this: `exit 137` means the VM ran out of memory. `colima` allocates 2Gb per VM, which apparently is not enough. 6Gb works.

Reply:
I have seen this particular error, too, but was never able to isolate it to anything local. If it is a local issue or a bug with the playground, it's likely internal to the contract-topos container in some way.

So, I left this one untouched in the docs and figured that it could be addressed in one way or another later.

Reply:
Addendum -- apparently one cause of this error is a topos-playground run that failed because of another error. I added a section to the troubleshooting, but, basically, for at least some instances of this error, just running a `topos-playground clean` will fix it.

Reply:
Ah, interesting.  In looking at my _similar_ error that was caused just by wonky state from a previous run, I had an exit 1.  This suggests a way, though, that the playground could be more helpful in the future, by providing specific feedback in response to particular exit codes, when there are meaningful exit codes. Let's leave it for a future PR, though. I'll file an issue right now, however, so that I don't forget it.